src/Heap/Traits/VisitorTrait.php

```diff
@@ -37,6 +37,6 @@ public function visited(string $branch): bool
      */
     public function markVisited(string $branch): void
     {
-        $this->visited[$branch] = null;
+        $this->visited[$branch] = true;
```

Comment:
Hello @wolfy-j, starting with this one...

SInce we mark something as true (been visited?) it makes more sense to me to use `true`, then using `!empty()` in (https://github.com/cycle/orm/pull/93/commits/6840df00f19a65fead24a3a84623d7c0d026e3ed#diff-e6c6997fe10fbc4dfb37444c86576d2fR30) is again faster than function call array_key_exists (at least for php<7.4)

Reply:
Good, I agree.